src/page/sinfo.rs

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::{Class, Flags};
+use core::arch::asm;
+use x86_64::structures::paging::Page;
```

Comment:
https://github.com/enarx/enarx/blob/main/STYLE.md#use-declarations

Reply:
I'm looking at the order and it looks correct. Is it missing new lines?